packages/instrumenter/src/parsers/svelte-parser.ts

```diff
@@ -0,0 +1,100 @@
+import { BaseNode, Program } from 'estree';
+import { parse as svelteParse, walk } from 'svelte/compiler';
+import { Ast as InternalSvelteAst } from 'svelte/types/compiler/interfaces.js';
+
+import { AstFormat, SvelteAst, SvelteRootNode, SvelteNode } from '../syntax/index.js';
+
+import { ParserContext } from './parser-context.js';
+
+const header = '<script></script>\n\n';
+
+export async function parse(text: string, fileName: string, context: ParserContext): Promise<SvelteAst> {
+  let ast = svelteParse(text);
```

Comment:
This will make the import lazy; thus will only be done when an _actual_ svelte file exists.
```suggestion
  const { parse as svelteParse } = await import ('svelte/compiler');
  let ast = svelteParse(text);
```